app/src/organisms/RobotSettingsDashboard/NetworkSettings/index.tsx

```diff
@@ -97,13 +98,22 @@ function NetworkSettingButton({
   networkName,
   onClick,
 }: NetworkSettingButtonProps): JSX.Element {
+  const PUSHED_STATE_STYLING = css`
```

Comment:
Maybe STYLE since other styles that are defined by `css` use `_STYLE`.

```suggestion
  const PUSHED_STATE_STYLE = css`
```